packages/twenty-front/src/modules/object-record/record-table/components/RemoveSortingModal.tsx

```diff
@@ -0,0 +1,52 @@
+import { useRecoilValue, useSetRecoilState } from 'recoil';
+
+import { removeSortingModalOpenState } from '@/object-record/record-table/states/removeSortingModalOpenState';
+import { ConfirmationModal } from '@/ui/layout/modal/components/ConfirmationModal';
+import { useCombinedViewSorts } from '@/views/hooks/useCombinedViewSorts';
+import { useGetCurrentView } from '@/views/hooks/useGetCurrentView';
+
+export const RemoveSortingModal = ({
+  recordTableId,
+}: {
+  recordTableId: string;
+}) => {
+  const { currentViewWithCombinedFiltersAndSorts } =
+    useGetCurrentView(recordTableId);
+
+  const viewSorts = currentViewWithCombinedFiltersAndSorts?.viewSorts || [];
+  const fieldMetadataIds = viewSorts.map(
+    (viewSort) => viewSort.fieldMetadataId,
+  );
+  const removeSortingModalOpen = useRecoilValue(removeSortingModalOpenState);
```

Comment:
is as a prefix would disambiguate a little bit, as we try to keep action verbs as first word for function naming.

Reply:
Plus it should be consistent with `setIsRemoveSortingModalOpen`.

Also you can use `useRecoilState` here instead of `useRecoilValue` + `useSetRecoilState`